.vitepress/config.mts

```diff
@@ -35,6 +35,333 @@ function nav(): DefaultTheme.NavItem[] {
+          collapsed: true,
+          items: [],
+        },
+        {
+          text: 'Advanced Filtering',
+          collapsed: true,
+          items: [],
+        },
+        {
+          text: 'Block Stream',
+          collapsed: true,
+          items: [
+            {
+              text: 'Subscribe to Block Stream',
+            },
+            {
+              text: 'View output',
+            },
+          ],
+        },
+      ],
+    },
+    {
+      text: 'Instructions',
+      collapsed: true,
+      items: [
+        {
+          text: 'Use Instructions',
+        },
+        {
+          text: 'Combine Instructions via Expressions',
+        },
+      ],
+    },
+    {
+      text: 'Metadata',
+      collapsed: true,
+      items: [
+        {
+          text: 'Set Key Value',
+        },
+        {
+          text: 'Remove Key Value',
+        },
+        {
+          text: 'Access Metadata',
+        },
+      ],
+    },
+    {
+      text: 'Peers',
+      collapsed: true,
+      items: [
+        {
+          text: 'Register and Unregister Peers',
+        },
+        {
+          text: "Check Peer's Load",
+        },
+        {
+          text: 'Find the Leader Among Running Peers',
+        },
+      ],
+    },
+    {
+      text: 'Queries',
+      collapsed: true,
+      items: [
+        {text: 'Use queries'},
+        {text: 'Filter query results'},
+        {text: 'Use Sorting and Pagination'},
+        {text: 'Query Connected Peers'},
+      ],
+    },
+    {
+      text: 'Telemetry',
+      collapsed: true,
+      items: [
+        {
+          text: 'Check Status',
+        },
+        {
+          text: 'Get Metrics',
+        },
+        {
+          text: 'Monitor Iroha Performance',
+        },
+        {
+          text: 'Check Health',
+        },
+      ],
+    },
+    {
+      text: 'Transactions',
+      collapsed: true,
+      items: [
+        {
+          text: 'Create Transactions',
+        },
+        {
+          text: 'Submit Transactions',
+        },
+        {
+          text: 'Compile Transactions',
+        },
+        {
+          text: 'Use Multi-Signature Transactions',
+        },
+      ],
+    },
+    {
+      text: 'Triggers',
+      collapsed: true,
+      items: [
+        {
+          text: 'Data Triggers',
+          collapsed: true,
+          items: [
+            {
+              text: 'Register a Data Trigger',
+            },
+          ],
+        },
+        {
+          text: 'Time Triggers',
+          collapsed: true,
+          items: [
+            {
+              text: 'Register a Scheduled Trigger',
+            },
+            {
+              text: 'Register a Pre-commit Trigger',
+            },
+          ],
+        },
+        {
+          text: 'By-call Triggers',
+          collapsed: true,
+          items: [
+            {
+              text: 'Register a By-call Trigger',
+            },
+          ],
+        },
+      ],
```

Comment:
```suggestion
      items: [
        {
            text: 'Register a Data Trigger',
        },
        {
            text: 'Register a Scheduled Trigger',
        },
        {
            text: 'Register a Pre-commit Trigger',
        },
        {
            text: 'Register a By-call Trigger',
        },
        {
            text: 'Unregister a By-call Trigger',
        },
      ],
```

how about we do it like this?

Reply:
I guess it depends on whether we want to provide examples for more than just registration of triggers.